app/views/pages/home.html.erb

```diff
@@ -1,12 +1,54 @@
-<% if current_user %>
-  <h1><%=t('welcome')%> <%= current_user.first_name %></h1>
-<% else %>
-  <div class="content-container">
-  <%= link_to new_user_path, class: "content-card" do %>
-  <%= t('create_user') %>
-  <% end %>
-  <div class="content-container">
-  <%= link_to new_user_session_path, class: "content-card" do %>
-  <%= t('login_user') %>
-  <% end %>
-<% end %>
+<div class="grid-container">
+  <div class="left-column">
+    <% if current_user %>
+      <h1><%= t('welcome') %> <%= current_user.first_name %></h1>
+      <h2><%= t('journal_entries_today') %></h2>
+      <% today_journal_entries = current_user.journal_entries.where(date: Date.today) %>
+      <div class="content-container">
+        <% today_journal_entries.each do |entry| %>
+          <div class="content-card">
+            <h3><%= entry.title %></h3>
+            <p><%= entry.content %></p>
+          </div>
+        <% end %>
+      </div>
+      <h2><%= t('goals_today') %></h2>
```

Comment:
Let's add the keys to the translation files

Reply:
thank you, i did